examples/00_additive_single_bead.py

```diff
@@ -1,13 +1,13 @@
-# Copyright (C) 2023 ANSYS, Inc. and/or its affiliates.
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
```

Comment:
@klmcadams I thought the Ansys copyright format required us to only use a single year, not a span. Did legal, or somebody else, change that?

Reply:
@pkrull-ansys The .reuse library that the add-license-header hook uses to create the headers makes it a span if there's already an existing year in the copyright statement. I can double check to make sure that's okay

Reply:
We can verify with legal just in case, but it should be allowed.

Reply:
@klmcadams @RobPasMue This was my bad. I was mistaken about the format, which is defined on pages 31-32 in https://ansys.sharepoint.com/:b:/r/sites/LegalSHARESite/Shared%20Documents/Intellectual%20Property/All%20Employee%20Access/IP%20Handbook%20-%20Current/IP%20Handbook%202021%20update_2022%20Jan%2025.pdf?csf=1&web=1&e=9VYsqT